CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Change Log
+
+## 2025-01-04
+
+During this week, Chase developed some early test code to simulate the location and reliability
+of the delivery agents. Spencer spent some time developing code that can generate orders on
+demand. When these two files get combined, they can generate data that looks something like the
+data provided to us by the client.
+
+We plan to use this data to experiment with various choice algorithms on how to choose between
+different subcontractors.
+
+In terms of next steps, we need to develop an algorithm for choosing these subcontractors and run
```

Comment:
Have you thought of trying a multi-armed bandit approach here? I've heard they are very compelling and very fun to code.